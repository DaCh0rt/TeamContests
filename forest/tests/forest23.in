251 339
7113 -6828
-4816 1489
-6812 -9349
-231 897
8038 703
-4600 7186
-763 -8474
6441 310
-352 -9080
8912 5015
7452 394
-4899 -1258
-9335 3874
3720 -8238
-7682 -6620
-9886 -5573
9072 -5514
-1611 2556
-1125 -7974
-8629 6696
8119 -8613
-9008 5837
-5657 -1434
56 -73
-3270 -1835
-5013 -6768
-3425 -9984
1981 8698
-7918 -6180
5797 -8131
-5211 3740
-2161 -3456
-1436 -7241
-1007 -2521
-1645 -365
8767 5923
-9301 -9611
-9545 -3191
-8703 -2499
8203 1593
-6474 -7540
-4460 -7220
-6421 6431
-3597 -4217
-9847 -508
5049 -3286
7715 1805
-9511 -47
1022 3081
3978 -3601
8108 -9458
-1126 -3771
-305 -8886
1472 -766
9510 -5779
5417 -3359
9836 9661
3876 2866
9652 -3637
-3714 1853
-5685 -4649
3138 -2437
9702 -8421
-8474 7886
-3608 4750
9344 -2379
2659 1806
-4747 8597
-7454 9559
8531 9224
2605 7293
-7432 -4176
-3835 -8193
-7624 7337
-630 -4101
-3070 -5748
9682 -8846
-4136 2640
-9893 2112
-5483 8265
-9563 -8740
-459 -487
-9576 7695
8450 -7918
6552 -2013
-7834 -4190
818 1687
-1824 -6891
7350 7419
6256 6965
6670 7727
1083 7353
-8802 8115
-4858 -7874
6211 3864
-732 7174
2810 -6749
7852 -4557
-2522 -658
671 -8508
-7793 8999
448 -8220
-5407 2595
6168 3803
1104 -2678
4910 9881
-341 1851
7548 6037
-3180 649
-5365 3375
4137 5073
-847 -5988
-9369 2971
-6147 -241
2586 -6530
9814 -507
1794 -8141
4935 -9113
8288 3780
5995 400
-6773 5108
6611 5866
9873 4349
-5982 -8487
9354 4144
8827 -2822
-2255 1818
5905 6780
-2334 -6907
-622 -9550
6329 8973
-7927 2049
9584 1849
-2043 5481
1837 4939
-6903 976
-4212 5097
2774 -9208
-9285 -5427
2749 1491
8859 -4743
7520 -5028
-5593 -7722
-8968 7957
3531 -5185
-6014 635
538 3702
-2249 -6198
-6459 -8736
6571 -5669
-6907 -3249
4152 8506
5721 4599
1281 -9095
3037 -7311
-897 9956
-5789 -5872
108 7830
-5805 4907
-3700 -5113
6342 932
8241 -522
4183 1593
-3975 1048
-6173 2300
-3769 5269
2936 -7635
2448 9
6001 -2366
7966 -3479
7968 8990
-3540 -6868
3598 6242
7671 -350
5740 9731
5169 -4746
-8515 3172
8413 -666
-4117 9823
-2128 -2908
-2090 3444
-5951 -8813
-3538 4114
-1966 -8134
9820 -3901
7639 -3358
-8572 2656
-7637 -6800
-2530 6688
-4085 5194
-5107 891
-7554 -2746
2061 5943
3082 -4308
-6270 2412
-8717 -6851
6465 2554
-9749 5894
-3999 6809
6016 7659
960 -9267
-4091 5753
8983 -1721
-9351 5020
492 1152
-8474 1458
-6004 5051
3338 -7469
-9000 -8640
-7181 -1413
-4463 2806
-8814 -6037
6555 -6377
-621 6490
1390 -9597
7637 6759
614 -4565
-641 -2560
-234 -7489
1951 -9706
-4112 -1129
4694 -3705
2051 4897
-23 -1534
-5386 -8330
-3314 5698
-8763 5734
5181 -6523
365 -1827
3944 -2874
-7567 -2149
-5237 -9278
-5496 -4367
-4505 1964
4132 2120
802 -5834
-3279 -455
-4948 3435
-2257 8031
278 8154
9847 -5578
5731 -1217
-2042 -9293
7648 5582
-8810 -8151
-3662 6119
7981 -596
3741 -8475
275 -1573
-932 -3228
5764 6270
