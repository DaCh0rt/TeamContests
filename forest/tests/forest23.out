77267.94 406083340.74
